habitat_sim/agent/controls/object_controls.py

```diff
@@ -16,9 +17,10 @@
 
 # epislon used to deal with machine precision
 EPS = 1e-5
+_3d_point = Union[np.ndarray, mn.Vector3, Tuple[float, float, float]]
 
 
-def _noop_filter(start: np.array, end: np.array) -> np.array:
+def _noop_filter(start: _3d_point, end: _3d_point) -> _3d_point:
```

Comment:
It seems like this noop filter was written to work with any type and is very general-purpose. Is there a good reason to narrow the types so specifically?

Reply:
Er, I guess you're making the type broader, not narrower here. I guess what I'm saying is, both the old typing (nd.array) and the new typing (_3d_point) seem unnecessarily narrow.